src/Console/LogVerbosityInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Copyright © 2017 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+namespace Infection\Console;
+
+interface LogVerbosityInterface
```

Comment:
just a question: should this "constants container" be an interface or a class?

Just didn't see such practice with interfaces that hold constants only without being implemented, but it looks interesting. Is there any read about this? :)

Reply:
I won't give you anything to read as this is not a good practice :) (When you interface contains constants only).

I'll create final class instead.